test/parallel/test-module-require-depth.js

```diff
@@ -1,16 +0,0 @@
-// Flags: --expose_internals
-'use strict';
-require('../common');
-const fixtures = require('../common/fixtures');
-const assert = require('assert');
-const {
-  requireDepth
-} = require('internal/modules/cjs/helpers');
-
```

Comment:
☝️ Test above was likely borked because it restructured the `requireDepth` property.